src/ProjectTemplates/Web.ProjectTemplates/content/StarterWeb-CSharp/Services/MicrosoftGraphServiceExtensions.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
I don't think we should be including copyright headers in templates right?

Reply:
Fair enough. We'll remove them. Thanks @JunTaoLuo